infra/lib/api-stack.ts

```diff
@@ -293,6 +293,14 @@ export class APIStack extends Stack {
       dynamoDBTokenTable,
     });
 
+    const withingsWebhookResource = api.root.addResource("withings");
```

Comment:
this needs to be `const withingsWebhookResource = webhookResource.addResource("withings");`